app/src/main/java/nz/org/cacophony/cacophonometer/Server.java

```diff
@@ -31,7 +31,8 @@
 
     private static final String TAG = Server.class.getName();
 
-    private static final String UPLOAD_AUDIO_API_URL = "/api/v1/audiorecordings";
+   // private static final String UPLOAD_AUDIO_API_URL = "/api/v1/audiorecordings";
```

Comment:
There's probably no need to keep this around (that's what version control is for)